Improves Monitoring VQE convergence

divshacker opened the pull request:
<!--
⚠️ If you do not respect this template, your pull request will be closed.
⚠️ Your pull request title should be short detailed and understandable for all.
⚠️ If your pull request fixes an open issue, please link to the issue.

✅ I have added the tests to cover my changes.
✅ I have updated the documentation accordingly.
✅ I have read the CONTRIBUTING document.
-->

### Summary
This PR will improve the tutorial [02_vqe_advanced_options](https://qiskit.org/documentation/tutorials/algorithms/02_vqe_advanced_options.html)


### Details and comments
fixes #1403 
Part of #1401 
I have updated Few old links and references and changed images to SVGs

woodsp-ibm commented on tutorials/algorithms/02_vqe_advanced_options.ipynb:
```diff
@@ -6,7 +6,7 @@
     "# Advanced VQE Options\n",
     "\n",
-    "In the first algorithms tutorial, you learned how to set up a basic [VQE](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.VQE.html) algorithm. Now, you will see how to provide more advanced configuration parameters to explore the full range of capabilities of Qiskit's variational algorithms: [VQE](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_ ...
+    "In the first algorithms tutorial, you learned how to set up a basic [VQE](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.VQE.html) algorithm. Now, you will see how to provide more advanced configuration parameters to explore the full range of capabilities of Qiskit's variational algorithms: [VQE](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_ ...
```
This was correct as it was before. It was intended to point to the `VQD` eigensolver as its similar to VQE but can produces a number of states (excited states) as well the ground state.

divshacker replied:
It was written in the documentation that VQD will be deprecated later and I also found the alternative so I replaced it.
I will revert that change. 
Thanks

woodsp-ibm replied:
Yes, for minimum(eigensolvers) there exist the now deprecated versions and new versions. If it has `(minimum)_eigensolvers` where there is no `_` in `eigensolvers` ie. old version is `(minumum_)eigen_solvers` then its the new one (If also its one of these types and points to the convenience import direct in qiskit.algorithms then its the old one - we had to keep it this way to avoid breaking existing code - at some point in the future, after the old ones are removed, we may be able to have these point at the new ones). Hopefully these tutorials are correct in that we just went through them recently to update them to primitives and the new algos - the above though may helpful as you look at others just in case!

woodsp-ibm commented:
> I have updated Few old links and references and changed images to SVGs

As far out the output cell image format - these notebooks re-run/converted to html by CI for the published docs.

divshacker commented:
> > I have updated Few old links and references and changed images to SVGs
> 
> As far as the output cell image format - these notebooks re-run/converted to html by CI for the published docs.

Yes, I know that the images format will be changed anyway before publishing but like after making changes and rerunning the tutorials locally the images are automatically changed to SVGs due to my local environment. I am not doing it intentionally. Also as these tutorials would also remain in GitHub repo so it's good to change them into SVGs if they are not in that format.

divshacker commented:
This PR is ready for review
Thanks

Eric-Arellano reviewed (approved):
Thanks!
